scan/src/bindings/BandWeb3.re

```diff
@@ -61,7 +69,17 @@ let broadcast = (instance, signedMsg) => {
   let%Promise rawResponse = instance->_broadcast(signedMsg);
 
   Promise.ret(
-    JsonUtils.Decode.{txHash: rawResponse |> at(["txhash"], string) |> Hash.fromHex},
+    Tx(
+      JsonUtils.Decode.{
+        txHash: rawResponse |> at(["txhash"], string) |> Hash.fromHex,
+        rawLog: rawResponse |> at(["raw_log"], string),
+        success:
+          switch (rawResponse |> optional(field("logs", list(log => log)))) {
```

Comment:
@sorawit Is there a better way to handle this case (for checking log is available)?

Reply:
Can't we do `rawResponse |> optional(field("logs", _ => ()) |> Belt_Option.isSome`?